MoltenVKShaderConverter/MoltenVKShaderConverterTool/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+################################################################################
+# MoltenVKShaderConverter Executable
+################################################################################
+
+# Find the source files in this directory. We prefer to use GLOB because it is
+# easier to maintain in conjunction with the Xcode project file.
+file(GLOB_RECURSE SOURCES "*.c" "*.cpp" "*.m" "*.mm")
+
+add_executable(MoltenVKShaderConverter_bin ${SOURCES})
```

Comment:
I had to add a `_bin` suffix to avoid name clash with the `MoltenVKShaderConverter` folder. I'll look into how to avoid this without changing the name. Maybe it's okay because we can change the name on install or copy to `Package`.